test/spec/todo/todo.spec.ts

```diff
@@ -46,31 +46,49 @@ describe("POST /todos", () => {
   });
 });
 
-describe("GET /todos/:id", () => {
-  it("should fetch a todo item if it exists and if id is valid mongo id", async () => {
-    const todoItem = await testAppContext.TodoItemRepository.save(
-      new TodoItem({ title: "Fetching an item" })
+describe("PUT /todos/:id", () => {
+  it("should return 200 if todo exists and title is validately true and 400 if title is empty or not a string else 404", async () => {
```

Comment:
Please make separate test block for each case.
Do not mix multiple tests in one `it` block.

Reply:
Updated